cellprofiler/__main__.py

```diff
@@ -69,7 +70,8 @@ def main(args=None):
 
     options, args = parse_args(args)
 
-    if any(options.pipeline_filename.startswith(protocol) for protocol in ('http', 'https', 'ftp')):
+    o = urlparse.urlparse(options.pipeline_filename)
```

Comment:
Need to add `if options.pipeline_filename:` around lines 73-81, or else NoneType error results.